tensorflow/tensorboard/plugins/projector/plugin.py

```diff
@@ -136,7 +136,7 @@ def get_plugin_handlers(self, run_paths, logdir):
   @property
   def configs(self):
     """Returns a map of run paths to `ProjectorConfig` protos."""
-    run_path_pairs = self.run_paths.items()
+    run_path_pairs = list(self.run_paths.items())
```

Comment:
Instead of converting to a `list` here, it might be more readable to remove the use of `list.append()` and replace L143 with:

```python
      run_path_pairs = [('.', self.logdir)]
```